imapautofiler/rules.py

```diff
@@ -181,3 +185,24 @@ def check(self, message):
         header_value = message.get(self._header_name, '').lower()
         self._log.debug('%r in %r', self._regex.pattern, header_value)
         return bool(self._regex.search(header_value))
+
+
+class Expired(Rule):
```

Comment:
After we talked yesterday I thought a bit about this name. Instead of calling it "expired", which implies that the message is likely to be deleted, let's call it something like "Age". I might want a rule that takes everything from my inbox that is older than 4 weeks and moves it to an archive folder, for example. The messages aren't "expired" in that case, they're just archived.

Reply:
Ok, will be done.